README.md

```diff
@@ -231,13 +125,107 @@ passed as environment variables to the test script. To see which implementations
 require client secrets, please check the implementation manifest within the
 [vc-test-suite-implementations](https://github.com/w3c/vc-test-suite-implementations/tree/main/implementations) library.
 
-### Docker Integration (TODO)
 
-We are presently working on implementing a new feature that will enable the
-use of Docker images instead of live endpoints. The Docker image that
-you provide will be started when the test suite is run. The image is expected
-to expose the API provided above, which will be used in the same way that
-live HTTP endpoints are used above.
+## Usage
+
+```js
+npm i
+```
+
+### Running Specific Tests
+
+This suite uses [`mocha.js`](https://mochajs.org) as the test runner.
+Mocha has [multiple options](https://mochajs.org/#command-line-usage) for filtering which tests run.
+
+For example, the snippet below uses `grep` to filter tests by name, and only runs one of the test suites.
+
+```bash
+mocha --grep '"specificProperty" test name' ./tests/10-specific-test-suite.js
+```
+
+### Testing Locally
+
+If you want to test a single implementation or endpoints running locally, you can
+copy `localImplementationsConfig.example.cjs` to `.localImplementationsConfig.cjs`
+in the root directory of the test suite.
+
+```bash
+cp localImplementationsConfig.example.cjs .localImplementationsConfig.cjs
+```
+
+This file must be a CommonJS module that exports an array of implementations
+(the format is identical to the on defined in
+[VC Test Suite Implementations](https://github.com/w3c/vc-test-suite-implementations?tab=readme-ov-file#usage)):
+
+```js
+// .localImplementationsConfig.cjs defining local implementations
+// you can specify a BASE_URL before running the tests such as:
+// BASE_URL=http://localhost:40443/zDdfsdfs npm test
+const baseUrl = process.env.BASE_URL || 'https://localhost:40443/id';
+module.exports = [{
+  name: 'My Company',
+  implementation: 'My Implementation Name',
+  // only this implementation will be run in the suite
+  issuers: [{
+    id: 'did:key:zMyKey',
+    endpoint: `${baseUrl}/credentials/issue`,
+    supportedEcdsaKeyTypes: ['P-256', 'P-384'],
```

Comment:
remember that issuers should be separated by keyType

Reply:
@aljones15 so you want an additional array element with the same URL (in this case), but with a different `supportedEcdsaKeyTypes` single element array?

If we only ever want one value there, then `supportedEcdsaKeyTypes` should change to being a sting (and we should document why/how to handle that).